app/views/movies/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Movie Show Page</h1>
+
+<h2>Title: <%= @movie.title %></h2>
+<p>Creation Year: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+
+<div id="actors-age">
+  <h3>Actors (Youngest to Oldest)</h3>
+  <% @actors.each do |actor| %>
+    <p><%= actor.name %> </p>
+    <p>Age: <%= actor.age %></p>
+  <% end %>
+  <p>Average Age of Actors: <%= @movie.average_actor_age.round(2) %></p>
+</div>
+
+<div id="add-actor-form">
+  <h3>Add Actor to Movie</h3>
+  <%= form_with url: "/movies/#{(@movie.id)}", method: :post, local: true do |form| %>
```

Comment:
This is a good start, I think your method and url don't agree with each other though. If we're using `:post` that means we're creating something. The URL should inform us what we're creating; your URL says we're creating a movie by a specific id.... which doesn't relaly make sense. Instead, maybe we use one of these paths: 
```
# 1 
/movies/:id/actors

# 2 
/actor_movies
```

If you choose # 2, you'd have to include a hidden field in your form for the `movie_id` (or use a query param).